cas-server-support-saml-idp/src/main/java/org/jasig/cas/support/saml/services/SamlIdPEntityIdValidationServiceSelectionStrategy.java

```diff
@@ -22,6 +22,7 @@
  */
 @Component("samlIdPEntityIdValidationServiceSelectionStrategy")
 public class SamlIdPEntityIdValidationServiceSelectionStrategy implements ValidationServiceSelectionStrategy {
+    private static final long serialVersionUID = -2718740284657155873L;
     protected transient Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['VARIABLE_DEF' should be separated from previous statement.](https://www.codacy.com/app/mmoayyed/cas/file/2508477822/issues/source?bid=3256115&fileBranchId=3273836#l26)